File: wizards/com/sun/star/wizards/fax/CGFaxWizard.py
from wizards.common.ConfigGroup import *
from wizards.fax.CGFax import CGFax

class CGFaxWizard(ConfigGroup):

    def __init__(self):
        self.cp_FaxType = int()
        self.cp_BusinessFax = CGFax()
        self.cp_PrivateFax = CGFax()

File: wizards/com/sun/star/wizards/fax/CGFaxWizard.java
package com.sun.star.wizards.fax;

import com.sun.star.wizards.common.ConfigGroup;

public class CGFaxWizard extends ConfigGroup {

    public int cp_FaxType;
    public CGFax cp_BusinessFax = new CGFax();
    public CGFax cp_PrivateFax = new CGFax();

}
